client/src/components/application/Examine/RequestInfoHeader.vue

```diff
@@ -827,9 +832,6 @@
         // if jurisdiction not required, clear the data (ie: BC)
         if (!this.jurisdiction_required) this.$store.commit('jurisdiction', null)
 
-        // if corp num not required, clear the data
-        if (!this.corp_num_required) this.$store.commit('corpNum', null)
```

Comment:
this is the actual place where the corpNum is prevented from being wiped.  The rules in the application will need to be contemplated in the future when MRAS functionality is added name-request so that they will correctly discern when the corpNum should be cleared but at this point removing it is harmless as it won't be displayed to the examiners unless the current rules come back true so it won't be displayed to them at unexpected times but will be retained when it was already set by namerequest.